src/sentry/api/event_search.py

```diff
@@ -1633,31 +1643,68 @@ def parse_function(field, match=None):
     )
 
 
+def count_required_arguments(function):
+    n = 0
+    for arg in function["args"]:
+        if arg.has_default:
+            break
+        n += 1
+    return n
+
+
+def count_total_arguments(function):
+    return len(function["args"])
+
+
+def validate_argument_count(field, function, arguments):
```

Comment:
Could we add some tests for this function? Especially since it has 4 well defined cases